credentials-manager.js

```diff
@@ -115,7 +119,7 @@ class CredentialsManager {
     let credentials;
 
     try {
-      credentials = await fs.readFile(path, 'utf-8')
+      credentials = fs.readFileSync(path, 'utf-8')
```

Comment:
I would prefer using `promisify` (https://nodejs.org/dist/latest-v8.x/docs/api/util.html) instead of the sync version. You can revert almost all changes and promisify fs at the top.

Reply:
ok, changed it to use promisify